package.json

```diff
@@ -31,7 +31,7 @@
     "build:watch": "tsc -w -p tsconfig.release.json",
     "lint": "eslint . --ext .ts,.tsx",
     "lint-fix": "eslint . --ext .ts,.tsx --fix",
-    "test": "jest --coverage --detectOpenHandles",
+    "test": "jest --coverage --detectOpenHandles --forceExit",
```

Comment:
When I added the DB connection, the test suite would hang until I added this line

Reply:
sounds good

Reply:
Did you try adding the db.close() to the test files? https://github.com/facebook/jest/issues/3602. I think --forceExit might have some issues later on. (if db.close() doesn't work locally I'm good to approve this)

Reply:
Working on this now!